spec/features/manage_submissions_spec.rb

```diff
@@ -1,19 +1,18 @@
 require "rails_helper"
 require_relative("../support/controller_macros")
+require_relative("../controllers/controllers_shared_context")
 include ControllerMacros
 
 RSpec.describe "manage submissions user flow", type: :feature do
   describe "click button", js: true do
     context "when user is Instructor" do
+      include_context "controllers shared context"
```

Comment:
nit: some inconsistency with whether `include_context` is inside the context block or the outer describe block, don't know if we want to standardize it or not

Reply:
It kind of depends on whether or not you want it defined within a certain scope, whether you want it defined across all contexts in a describe block or within a certain context.